Remotion/Web/Development.WebTesting/Utilities/RetryUntilTimeout.cs

```diff
@@ -55,27 +55,27 @@ public void Run ()
       _retryUntilTimeout.Run();
     }
 
-    public static void Run ([NotNull] Action action)
+    public static void Run ([NotNull] Action action, TimeSpan? timeout = null, TimeSpan? retryInterval = null)

```

Comment:
given the extra parameters, this now requires xmldoc. the doc for the parameters (or the remarks) must also explain, how the timeout and the retryinterval interact.